### PR TITLE
Small corrections (superfluos spaces, missing command, ignore-rule)

### PR DESCRIPTION
**Pull request by brogon**

### CODE REVIEW

File: 03-serverspec/README.md [hunk 1]
```diff
@@ -1,3 +1,14 @@
+# Installation
+
+Just run
+```bash
+mvn install
+```
+in this directory to install the parent `pom.xml` used for docker builds.
```
Comment (cruwe) [inline 03-serverspec/README.md:7]: "in this directory" is not correct. The parent pom resides in 00-. What was your orignal intent? May I drop this safely?

**Review by cruwe — COMMENTED** [review]
Anyhow, thank you very much for spotting and repairing my fat fingers!

File: 03-serverspec/README.md (hunk 1 above)
Comment (brogon) [reply 03-serverspec/README.md:7]: I needed to call "mvn install" for the following commands to work. The rest is a copy'n'paste-error. :)

Comment (cruwe) [reply 03-serverspec/README.md:7]: ah ... now I get it. Completely escaped me that I never lost a word there ... I"ll need to fix two smallish issues after merging, but thank you very much for spotting that!

**Pull request merged**